### ISSUE TITLE
1. Event edit can also be moved to settings -> `Event`

### ISSUE THREAD

**Issue by fvacek** [Quick-Event/quickbox#801]
1. Event edit can also be moved to settings -> `Event`
2. I also think that `Check type - Classic race | Free order race` should be moved into this form
![image](https://user-images.githubusercontent.com/24588500/187397605-6cc8948b-0a99-4d16-9c42-85341bc28301.png)

_Originally posted by @otahirs in https://github.com/Quick-Event/quickbox/issues/780#issuecomment-1231388611_

**Comment by lukaskett** [comment]
related dialog when moving to the settings
<img width="554" height="154" alt="Image" src="https://github.com/user-attachments/assets/ffe5b632-a7d4-4ee3-959e-9c06d7b8788e" />

**Comment by sergiogpajares** [comment]
I want to make a UI proposal. 

**Edit Stage**
I suggest integrating stage configuration directly into the event creation/edit dialog. Instead of configuring stages separately, users would define all stage-related details as part of the initial setup.


<img width="638" height="532" alt="Image" src="https://github.com/user-attachments/assets/c0066b57-a8f7-4ebe-a3d0-ce71347037b9" />

I think this approach has the following benefits:
 - Configuring stages during event setup feels more natural and reduces context switching. Users don’t need to navigate through multiple screens to edit each stage.
 - The global event date/time field is removed. In a single-stage event, the event time is inherently the stage time. In a multi-stage event, a single global time becomes  unnecessary.

I took this opportunity to add a name to the stages, so it can be shown on the reports. This is intended to guide user when the result for a single stage are published on a multistage event. It can be ignored in single stages.

**Check type**
I suggest moving the check type configuration to the course level. The check type is something inherent to the course. For me,  it conceptually belongs there rather than being handled elsewhere in the workflow. I know it is unlikely that on the same competition you get one free order course and another one is linear. 

I would implement a selectable `course type` that can either be `linear` or `free order`. Defaults to `linear`.

<img width="1042" height="493" alt="Image" src="https://github.com/user-attachments/assets/08dc41ad-b060-4137-beb2-b53c32cf28ac" />

**Comment by lukaskett** [comment]
> I want to make a UI proposal.
> 
> **Edit Stage** I suggest integrating stage configuration directly into the event creation/edit dialog. Instead of configuring stages separately, users would define all stage-related details as part of the initial setup.
> 
> I think this approach has the following benefits:
> 
> * Configuring stages during event setup feels more natural and reduces context switching. Users don’t need to navigate through multiple screens to edit each stage.
> * The global event date/time field is removed. In a single-stage event, the event time is inherently the stage time. In a multi-stage event, a single global time becomes  unnecessary.
> 
> I took this opportunity to add a name to the stages, so it can be shown on the reports. This is intended to guide user when the result for a single stage are published on a multistage event. It can be ignored in single stages.
> 

Good idea. Setting active stage could be also controlled from the table and this row could be removed:

<img width="311" height="79" alt="Image" src="https://github.com/user-attachments/assets/09efe885-f399-40ed-83a0-65c94e61cd23" />

> **Check type** I suggest moving the check type configuration to the course level. The check type is something inherent to the course. For me, it conceptually belongs there rather than being handled elsewhere in the workflow. I know it is unlikely that on the same competition you get one free order course and another one is linear.
> 
> I would implement a selectable `course type` that can either be `linear` or `free order`. Defaults to `linear`.
> 

It's not related to this issue. Check type is part of `Settings -> tab Card reader`. @sergiogpajares please create a new issue.

<img width="576" height="513" alt="Image" src="https://github.com/user-attachments/assets/2c55d075-c723-438d-a8a4-2bd0880a4fe0" />

**Comment by sergiogpajares** [comment]
@lukaskett 

Personally, I would not move the current stage selector from its current location.

In my opinion, the stage selector has nothing to do with the event’s configuration. You typically set up the event once and rarely return to it, so I wouldn’t intuitively look for it there. Additionally, placing it in that section would introduce a lot of visual noise due to the number of options, making the interface feel less simple.

If your goal is to remove the stage button, it’s worth noting that it currently appears twice in the interface, as shown in the image.


<img width="711" height="484" alt="Image" src="https://github.com/user-attachments/assets/f695a48f-98f5-4d9a-a733-b9838cc92c84" />

The bottom display is not clickable, so one option could be to remove the top button and move the selector to the bottom. However, removing the top one might make it harder for users to find, which is a trade-off I wouldn’t recommend.

**Comment by fvacek** [comment]
> [@lukaskett](https://github.com/lukaskett)
> 
> Personally, I would not move the current stage selector from its current location.
> 
> In my opinion, the stage selector has nothing to do with the event’s configuration. You typically set up the event once and rarely return to it, so I wouldn’t intuitively look for it there. Additionally, placing it in that section would introduce a lot of visual noise due to the number of options, making the interface feel less simple.
> 
> If your goal is to remove the stage button, it’s worth noting that it currently appears twice in the interface, as shown in the image.
> <img alt="Image" width="711" height="484" src="https://private-user-images.githubusercontent.com/49535041/583606551-f695a48f-98f5-4d9a-a733-b9838cc92c84.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODQyMDEyNDIsIm5iZiI6MTc4NDIwMDk0MiwicGF0aCI6Ii80OTUzNTA0MS81ODM2MDY1NTEtZjY5NWE0OGYtOThmNS00ZDlhLWE3MzMtYjk4MzhjYzkyYzg0LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzE2VDExMjIyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyOGM2NDcyMzE2ODI1OWU0MDg2NDQwOTExZmVmMzg1ZWNhZDVkMmRjZjVkYzg1NGI2NTY2NjM3NGY5ZjBkYjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRmpwZWcifQ.c9kuVd14kgdNHNQGL3va6JLFK3HMMqynFP7X-7FTtXw">
> 
> The bottom display is not clickable, so one option could be to remove the top button and move the selector to the bottom. However, removing the top one might make it harder for users to find, which is a trade-off I wouldn’t recommend.

On the other hand, the current stage selector is only one control on toolbar, moving this to app menu (for example) can make bigger space for event data.
